src/lib/authserver/ui/new_krb_inst.rb

```diff
@@ -153,28 +153,28 @@ def ok_handler
     out, ok = ldap.create_person(kdc_dn_prefix, 'Kerberos KDC Connection', dir_suffix)
     MITKerberos.append_to_log('%s' % out)
     if !ok
-      Popup.Error(_('Failed to create Kerberos KDC connection user! Log output may be found in /var/log/YaST/y2log'))
+      Popup.Error(_('Failed to create Kerberos KDC connection user! Log output may be found in /var/log/YaST2/y2log'))
```

Comment:
Please, move path out of translation as you already did in the _new\_dir\_inst.rb_ file

```suggestion
      Popup.Error(format(_('Failed to create Kerberos KDC connection user! Log output may be found in %{y2log}'), y2log: y2log))
```

Reply:
Note: you have to create the `y2log` method here too to make it work, unless you decide to put it in some common included file.